channelDiff.go

```diff
@@ -0,0 +1,89 @@
+package main
+
+import (
+	"log"
+	"fmt"
```

Comment:
Put a blank line after this import. I prefer to organize imports like

```
import (
    "std/lib/stuff"
    "std/lib/stuff"

    "3rd/party/stuff"
    "3rd/party/stuff"

    "our/internal/packages"
    "our/internal/packages"
)
```

Reply:
I like it!